crates/ty_python_semantic/resources/mdtest/generics/specialize_constrained.md

```diff
@@ -142,12 +164,21 @@ def constrained[T: (Base, Unrelated)]():
     # revealed: None
     reveal_type(generic_context(constrained).specialize_constrained(ConstraintSet.always()))
     # revealed: None
+    reveal_type(generic_context(constrained).specialize_constrained(ConstraintSet.range(Never, T, object)))
+    # revealed: None
+    reveal_type(generic_context(constrained).specialize_constrained(ConstraintSet.range(Never, T, Any)))
```

Comment:
These are `None` because we have no basis for picking between `Base` and `Unrelated`, so we'll just fall back to `Unknown`?